tracer/test/Datadog.Trace.ClrProfiler.IntegrationTests/CI/XUnitEvpTestsV3.cs

```diff
@@ -0,0 +1,536 @@
+// <copyright file="XUnitEvpTestsV3.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+using System.Collections.Generic;
+using System.Linq;
+using System.Runtime.CompilerServices;
+using System.Threading.Tasks;
+using Datadog.Trace.Ci.Ipc;
+using Datadog.Trace.Ci.Ipc.Messages;
+using Datadog.Trace.Ci.Tags;
+using Datadog.Trace.Configuration;
+using Datadog.Trace.ExtensionMethods;
+using Datadog.Trace.Telemetry;
+using Datadog.Trace.TestHelpers;
+using Datadog.Trace.TestHelpers.Ci;
+using Datadog.Trace.Vendors.Newtonsoft.Json;
+using FluentAssertions;
+using VerifyXunit;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Datadog.Trace.ClrProfiler.IntegrationTests.CI;
+
+#if NET6_0_OR_GREATER
+[UsesVerify]
+public class XUnitEvpTestsV3 : TestingFrameworkEvpTest
```

Comment:
I'm guessing there's a _lot_ of duplication here with the xunit tests, right? Rather than duplicate 500 lines of test code, can we derive one from the other or something? FWIW, it  feels like a huge number of the asserts are essentially already covered by the snapshots, no? 🤔